File: Ejemplos/Comunicacion/Java/Cliente.java
import java.net.ServerSocket;
import java.net.Socket;
import java.io.DataOutputStream;
import java.io.IOException;

class Cliente 
{
	private static final int PUERTO = 5000;
	
	public static void main(String[] args) throws IOException
	{
		Socket cs = new Socket( "127.0.0.1", PUERTO );
		
		DataOutputStream al_servidor;
						
		al_servidor = new DataOutputStream(cs.getOutputStream());
		al_servidor.writeUTF( "Atendeme" );

		cs.close();	
	}
}

File: Ejemplos/Comunicacion/Python/cliente.py
import socket

PUERTO = 5000

def cliente():
    cs = socket.socket()

    try:
        cs.connect(("127.0.0.1", PUERTO))
    except ConnectionRefusedError:
        print("Error: no se pudo conectar al servidor.")
        return

    print("Conectado al servidor. Escribe 'fin' para salir.")
    while True:
        envie = input("> ").strip()
        cs.send(envie.encode("utf-8"))
        if envie == "fin":
            break

    print("Conexión cerrada.")
    cs.close()

if __name__ == '__main__':
    cliente()
